### PR TITLE
🌱 GitLab: Add RepoHost tag to OpenCensus view

### PR DESCRIPTION
**Pull request by raghavkaul**
#### What kind of change does this PR introduce?
This PR adds the RepoHost tag to the opencensus view for metrics. This should allow us to break down error rates/runtimes in cron monitoring by github.com/gitlab.com repos.

### CODE REVIEW

File: checker/check_runner.go [hunk 1]
```diff
@@ -100,6 +100,12 @@ func (r *Runner) Run(ctx context.Context, c Check) CheckResult {
 	if err != nil {
 		panic(err)
 	}
+
+	ctx, err = tag.New(ctx, tag.Upsert(stats.RepoHost, r.CheckRequest.Repo.Host()))
+	if err != nil {
+		panic(err)
```
Comment (spencerschrock) [inline checker/check_runner.go:106]: Lets avoid `panic`ing. Can you change the one above too? You can find a non-`panic` approach in `logStats`

File: checker/check_runner.go [hunk 2]
```diff
@@ -98,8 +98,14 @@ func (r *Runner) Run(ctx context.Context, c Check) CheckResult {
 
 	ctx, err := tag.New(ctx, tag.Upsert(stats.CheckName, r.CheckName))
 	if err != nil {
-		panic(err)
+		r.CheckRequest.Dlogger.Debug(&LogMessage{Text: fmt.Sprintf("tag.New: %v", err)})
 	}
+
+	ctx, err = tag.New(ctx, tag.Upsert(stats.RepoHost, r.CheckRequest.Repo.Host()))
+	if err != nil {
+		r.CheckRequest.Dlogger.Debug(&LogMessage{Text: fmt.Sprintf("tag.New: %v", err)})
```
Comment (spencerschrock) [inline checker/check_runner.go:106]: I'm not sure the cron will see debug statements, do we just want to return these as error like we do in `logStats` above?

Comment (raghavkaul) [reply checker/check_runner.go:106]: I could change that to a warning, but I don't see logging failure as a reason the entire checkrun should fail.

Comment (spencerschrock) [reply checker/check_runner.go:106]: can we log in a way that doesn't associate it with the the check request? it's not really a detail of the check either

**Review by spencerschrock — APPROVED** [review]
Thanks

**Pull request merged**